### PR TITLE
fix polyfill

### PR DESCRIPTION
**Pull request by gmarinov**
note PR base branch

further fixes:
blackscreen in Hungry Spirits (render-to-texture plugin and Three messing up the canvas)

tested in:

- desktop: Safari, Firefox, Chrome
- mobile: iOS iPhone 6S, iPhone SE, Android Samsung S6
- embed: all of the above
- for orientation: landscape and portrait on standalone and embedded
- for devicemotion: all mobile, standalone and embedded

needs further testing in MS Edge

notes:

- the polyfill now implements its own method to receive messages in iframe, which seems to be missing orientationchange processing. Switching orientation in an iframe didn't work for me. Additionally, with the latest version of Android, devicemotion and orientation events seem to need proxying just as on iOS.
- restored event on VR instructions showing and hiding
- restored "canInitiateRotation" method as "shouldRotateStart" in both MouseKeyboardDisplay and TouchPanner
- adjusted iOS minimum time delta constant (accelerometer) that previously flooded the console
- this may break backwards compatibility with some experiences (though those wouldn't work in new browsers anyway)

known issue: in Safari 11 on macOS High Sierra, links in the Share dialogue may not work. Should this happen, resizing the window with the panel onscreen restores the correct behaviour (‽).

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -271,10 +271,10 @@ app.get('/', function(req, res, next) {
 		case '360.vizor.io':
 		case '360.vizor.lol':
 			return threesixtyController.index(req, res, next)
-		case 'patches.vizor.io':
-			return homeController.index(req, res, next)
-		default:
+		case 'vizor.io':
```
Comment (kschzt) [inline app.js]: not needed anymore
